src/components/morphine-ui/Navigation/Topbar/TopbarResp1/TopbarResp1.jsx

```diff
@@ -0,0 +1,62 @@
+import './TopbarResp1.css';
+import { FaBars } from 'react-icons/fa';
+import { BtnIcon } from '../../../Button';
+import {
+  // topbarResp,
```

Comment:
remove commented code. that will make code more readable and look it will even better

Reply:
done